Incompatible with winit 0.28.x

winit 0.28.x introduced some breaking changes that broke egui_winit_platform:
```
error: Please select a feature to build for unix: `x11`, `wayland`
  --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/platform_impl/linux/mod.rs:10:1
   |
10 | compile_error!("Please select a feature to build for unix: `x11`, `wayland`");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0392]: parameter `T` is never used
  --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/event_loop.rs:35:22
   |
35 | pub struct EventLoop<T: 'static> {
   |                      ^ unused parameter
   |
   = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`

error[E0392]: parameter `T` is never used
  --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/event_loop.rs:46:34
   |
46 | pub struct EventLoopWindowTarget<T: 'static> {
   |                                  ^ unused parameter
   |
   = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`

error[E0392]: parameter `T` is never used
   --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/event_loop.rs:350:27
    |
350 | pub struct EventLoopProxy<T: 'static> {
    |                           ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`

error[E0392]: parameter `T` is never used
   --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/platform_impl/linux/mod.rs:635:20
    |
635 | pub enum EventLoop<T: 'static> {
    |                    ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`

error[E0392]: parameter `T` is never used
   --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/platform_impl/linux/mod.rs:642:25
    |
642 | pub enum EventLoopProxy<T: 'static> {
    |                         ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`

error[E0392]: parameter `T` is never used
   --> /home/<redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.27.5/src/platform_impl/linux/mod.rs:769:32
    |
769 | pub enum EventLoopWindowTarget<T> {
    |                                ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`
    = help: if you intended `T` to be a const parameter, use `const T: usize` instead

For more information about this error, try `rustc --explain E0392`.
error: could not compile `winit` due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
```
... Not sure where to begin to fix this! Maybe updating the dep on winit would suffice?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,8 +10,8 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 
 [dependencies]
-egui = { version = "0.20", default-features = false }
-winit = { version = "0.27", default-features = false }
+egui = { version = "0.21", default-features = false }
+winit = { version = "0.28" }
 copypasta = { version = "0.8", optional = true }
 webbrowser = { version = "0.8", optional = true }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -323,6 +323,7 @@ impl Platform {
                                         key,
                                         pressed,
                                         modifiers: winit_to_egui_modifiers(self.modifier_state),
+                                        repeat: false,
                                     });
                                 }
                             }

